queue_services/entity-emailer/src/entity_emailer/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.95.0'  # pylint: disable=invalid-name
+__version__ = '2.96.0'  # pylint: disable=invalid-name
```

Comment:
if the intention is to have this go out with this weeks release, please leave this at 2.95.0